tox.ini

```diff
@@ -1,13 +1,14 @@
 [tox]
 minversion = 1.8
-envlist = py{27}-ansible{23,24,25}
+envlist = py{27}-ansible{23,24,25,26}
 skipsdist = true
 
 [travis:env]
 ANSIBLE=
   2.3: ansible23
```

Comment:
Ansible 2.3 is EOL

Reply:
I know lot of environments that still use Ansible 2.3. I would keep it until it is officially decommissioned to know when and if we introduce features not compatible anymore with this release.

Reply:
It is officially decomissioned. That's what EOL stands for.